main.cpp

```diff
@@ -416,47 +346,52 @@ int main()
     
     Artist metallica;
     Artist::Album ride_the_lightning;
-    std::cout << ride_the_lightning.totalAlbumLength() << std::endl;
+    std::cout << "Total album length: " << ride_the_lightning.totalAlbumLength() << "\n" << std::endl;
 
     Animal dog;
+    std::cout << "Is animal Bipedal?" << std::endl;
     if(dog.isAnimalBipedal() == true)
     {
-         std::cout << "Bipedal" << std::endl;
+         std::cout << "Bipedal \n" << std::endl;
     }
     else
     {
-        std::cout << "Not Bipedal" << std::endl;
+        std::cout << "Not Bipedal \n" << std::endl;
     }
+    dog.classifyAnimal();
 
     Student philip;
-    std::cout << philip.calculateGPA() << std::endl;
+    philip.calculateGPA();
 
     Guitar gibson;
     Guitar::Strings ernie;
     gibson.restringGuitar(ernie);
 
     Airliner airbus737;
-    std::cout << airbus737.checkFuelAmount() << std::endl;
+    airbus737.checkFuelAmount();
 
     Iphone s5;
+    std::cout << "Can Iphone run new OS?" << std::endl;
     if(s5.canRunNewOS() == true)
```

Comment:
This function returns true or false. you don't need to compare it with `true` via the `==`.
`if( s5.canRunNewOS() )` is all you need.
- [ ] revise.